Your_first_ASDF_converter.ipynb

```diff
@@ -0,0 +1,554 @@
+    "lists and dictionaries), as well as objects that have \n",
+    "serialization defined (e.g., astropy models and GWCS).\n",
+    "For example, if you define a class that has as one of\n",
+    "its attributes another Python class instance that doesn't\n",
+    "have a converter defined, the conversion will fail.\n",
+    "\n",
+    "Goal of this tutorial\n",
+    "----------------------\n",
+    "\n",
+    "There are several different ways of adding converters, some\n",
+    "simpler than this tutorial. This tutorial, however, expects\n",
+    "that the user of the ASDF extension wants to do as little\n",
+    "extra work to use the the extension to ASDF, ideally\n",
+    "none at all, and this means using entry points. \n",
+    "Furthermore, although much of this particular case can be\n",
+    "placed in fewer files, there is a good reason to use the \n",
+    "separate files used in this tutorial.\n",
+    "\n",
+    "Groundwork\n",
+    "--------------\n",
+    "\n",
+    "For ASDF to be aware of the converters, it is necessary \n",
+    "to include in the setup.cfg file information for an entry\n",
+    "point for ASDF. Entry points are\n",
+    "a very useful Python tool for making plug-ins for packages\n",
+    "easy for users of the plug-in to use, both in the installation\n",
+    "and usage aspect.\n",
+    "Entry points remove the need for the core package to be continually \n",
+    "updated with new extension packages that it has to be aware of.\n",
+    "\n",
+    "This information is provided in the converter package's\n",
+    "setup.cfg (it could be in\n",
+    "setup.py, but the .cfg file is the usual place to put this\n",
+    "information). What happens when the package is installed is\n",
+    "that information about entry points is saved by Python. Python\n",
+    "provides an API to the core package for it to discover what \n",
+    "entry points have been designated for that package so that it can\n",
+    "make use of them.\n",
+    "\n",
+    "In this example we will bundle the converters with the class\n",
+    "definitions that the converters will serialize. It is not\n",
+    "required that the class definitions must be in the same\n",
+    "package as the converters. For simplicity, they are for this\n",
+    "tutorial.\n",
+    "\n",
+    "We will name the converter package mfconverter (for My First\n",
+    "Converter) so this will create a package of that name in\n",
+    "the current directory. This tutorial will have the bare bones\n",
+    "files needed for that. The following will generate\n",
+    "such a barebones directory structure.\n",
+    "\n",
+    "Since this tutorial effectively is editing files and needs\n",
+    "to move between different directories, it is more awkward than\n",
+    "usual for a notebook\n",
+    "\n",
+    "Required Software installed\n",
+    "--------------------------------\n",
+    "\n",
+    "numpy\n",
```

Comment:
This is rendering funny:

<img width="348" alt="image" src="https://user-images.githubusercontent.com/2895013/122963809-9d050e80-d354-11eb-8e71-b03bff2c60c7.png">

Reply:
You don't like funny? I made them a list as I did in the second tutorial.